### DISCUSSION TITLE
Cacher un acteur d'une scène suivant le niveau d'accès de l'utilisateur

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#593] [Q&A]
Bonjour,

Dans mon menu, j'ai créé un Additionnel (UserSessionLevel) sur l'acteur que je veux rendre non visible si la session de l'utilisateur n'est pas de niveau "administrateur".

Je voudrais transférer mon Additionnel sur le premier acteur Empilement de ma scène (que j'ai nommé "root") pour simplifier le synapps car j'ai d'autres acteurs à cacher avec la même condition.

Quel est la synthaxe à mettre pour changer mon évènement OnInit de l'acteur afin de pointer l'additionnel d'un autre acteur ? 

Voici mon script actuel sur l'acteur :
`if (this.additionals.UserSessionLevel < 4 ) {
  this.properties.visible = 0
} else {
  this.properties.visible = 1
}`

Merci d'avance

**Comment by Raphwit** [comment]
Bonjour ,

Est-ce que cet acteur et votre menu son contenu dans une seule et même scène ou sont-ils dans deux scènes différentes ?
S'ils sont dans des scènes différentes, est-ce que vous utilisez un acteur écran ?

**Reply by JeremieDENAEGEL** [reply]
Bonjour,

Il s'agit d'acteurs dans la même scène. Cela me semble plus cohérent suivant la structure de mon synapps.
